[2025-05-28T09:47:07.305+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: animal_etl_pipeline_v1.0.1.load_animals scheduled__2025-05-27T02:00:00+00:00 [queued]>
[2025-05-28T09:47:07.318+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: animal_etl_pipeline_v1.0.1.load_animals scheduled__2025-05-27T02:00:00+00:00 [queued]>
[2025-05-28T09:47:07.318+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2025-05-28T09:47:07.336+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): load_animals> on 2025-05-27 02:00:00+00:00
[2025-05-28T09:47:07.342+0000] {standard_task_runner.py:60} INFO - Started process 73 to run task
[2025-05-28T09:47:07.345+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'animal_etl_pipeline_v1.0.1', 'load_animals', 'scheduled__2025-05-27T02:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/animal_etl_dag.py', '--cfg-path', '/tmp/tmp84tfn04o']
[2025-05-28T09:47:07.346+0000] {standard_task_runner.py:88} INFO - Job 16: Subtask load_animals
[2025-05-28T09:47:07.406+0000] {task_command.py:423} INFO - Running <TaskInstance: animal_etl_pipeline_v1.0.1.load_animals scheduled__2025-05-27T02:00:00+00:00 [running]> on host 92026f8f66a7
[2025-05-28T09:47:07.490+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='admin@company.com' AIRFLOW_CTX_DAG_OWNER='data_team' AIRFLOW_CTX_DAG_ID='animal_etl_pipeline_v1.0.1' AIRFLOW_CTX_TASK_ID='load_animals' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-27T02:00:00+00:00'
[2025-05-28T09:47:07.504+0000] {logging_mixin.py:188} INFO - Processing 5 batches
[2025-05-28T09:47:07.505+0000] {logging_mixin.py:188} INFO - Loading batch 1/5 with 13 animals
[2025-05-28T09:47:07.505+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:08.538+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:10.567+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:10.598+0000] {logging_mixin.py:188} INFO - ‚ùå Batch 1 failed with error: RetryError[<Future at 0x7f043c861520 state=finished raised ExternalAPIException>]
[2025-05-28T09:47:10.598+0000] {logging_mixin.py:188} INFO - Loading batch 2/5 with 13 animals
[2025-05-28T09:47:10.599+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:11.626+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:13.656+0000] {animals_api_hook.py:129} INFO - Sending 13 animals to home endpoint
[2025-05-28T09:47:13.687+0000] {logging_mixin.py:188} INFO - ‚ùå Batch 2 failed with error: RetryError[<Future at 0x7f043c869bb0 state=finished raised ExternalAPIException>]
[2025-05-28T09:47:13.688+0000] {logging_mixin.py:188} INFO - Loading batch 3/5 with 11 animals
[2025-05-28T09:47:13.688+0000] {animals_api_hook.py:129} INFO - Sending 11 animals to home endpoint
[2025-05-28T09:47:14.714+0000] {animals_api_hook.py:129} INFO - Sending 11 animals to home endpoint
[2025-05-28T09:47:16.749+0000] {animals_api_hook.py:129} INFO - Sending 11 animals to home endpoint
[2025-05-28T09:47:16.779+0000] {logging_mixin.py:188} INFO - ‚ùå Batch 3 failed with error: RetryError[<Future at 0x7f043c861df0 state=finished raised ExternalAPIException>]
[2025-05-28T09:47:16.780+0000] {logging_mixin.py:188} INFO - Loading batch 4/5 with 10 animals
[2025-05-28T09:47:16.780+0000] {animals_api_hook.py:129} INFO - Sending 10 animals to home endpoint
[2025-05-28T09:47:17.805+0000] {animals_api_hook.py:129} INFO - Sending 10 animals to home endpoint
[2025-05-28T09:47:19.835+0000] {animals_api_hook.py:129} INFO - Sending 10 animals to home endpoint
[2025-05-28T09:47:19.863+0000] {logging_mixin.py:188} INFO - ‚ùå Batch 4 failed with error: RetryError[<Future at 0x7f043c871310 state=finished raised ExternalAPIException>]
[2025-05-28T09:47:19.864+0000] {logging_mixin.py:188} INFO - Loading batch 5/5 with 6 animals
[2025-05-28T09:47:19.864+0000] {animals_api_hook.py:129} INFO - Sending 6 animals to home endpoint
[2025-05-28T09:47:20.888+0000] {animals_api_hook.py:129} INFO - Sending 6 animals to home endpoint
[2025-05-28T09:47:22.918+0000] {animals_api_hook.py:129} INFO - Sending 6 animals to home endpoint
[2025-05-28T09:47:22.945+0000] {logging_mixin.py:188} INFO - ‚ùå Batch 5 failed with error: RetryError[<Future at 0x7f043c877760 state=finished raised ExternalAPIException>]
[2025-05-28T09:47:22.946+0000] {logging_mixin.py:188} INFO - üìä Load Summary: {'status': 'completed', 'total_batches': 5, 'successful_batches': 0, 'failed_batches': 5, 'total_animals': 53}
[2025-05-28T09:47:22.946+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'completed', 'total_batches': 5, 'successful_batches': 0, 'failed_batches': 5, 'total_animals': 53}
[2025-05-28T09:47:22.988+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=animal_etl_pipeline_v1.0.1, task_id=load_animals, execution_date=20250527T020000, start_date=20250528T094707, end_date=20250528T094722
[2025-05-28T09:47:23.016+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T09:47:23.046+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
